File: RxUI/src/main/java/com/tamsiree/rxui/view/progressing/animation/interpolator/Ease.java
package com.tamsiree.rxui.view.progressing.animation.interpolator;

import android.view.animation.Interpolator;

/**
 * @author tamsiree
 */
public class Ease {
    public static Interpolator inOut() {
        return PathInterpolatorCompat.create(0.42f, 0f, 0.58f, 1f);
    }
}

File: RxUI/src/main/java/com/tamsiree/rxui/view/progressing/animation/interpolator/Ease.kt
package com.tamsiree.rxui.view.progressing.animation.interpolator

import android.view.animation.Interpolator

/**
 * @author tamsiree
 */
object Ease {
    @JvmStatic
    fun inOut(): Interpolator {
        return PathInterpolatorCompat.create(0.42f, 0f, 0.58f, 1f)
    }
}